build-system/tasks/visual-diff/index.js

```diff
@@ -521,10 +521,13 @@ async function snapshotWebpages(percy, browser, webpages) {
     await sleep(WAIT_FOR_TABS_MS);
   }
   log('travis', '\n');
-  if (isTravisBuild()) {
+  if (isTravisBuild() && testErrors.length > 0) {
     testErrors.sort((a, b) => a.name.localeCompare(b.name));
-    // TODO(danielrozenberg): add Travis log folding.
+    log('info', colors.yellow('Tests warnings and errors:'),
+        'expand this section');
+    exec('echo travis_fold:start:lint_results && echo');
```

Comment:
The string after `travis_fold:start:` must match the string after `travis_fold:end:`

Reply:
That explains why this isn't working for me >__<